client/src/legacy/LeftAndMain.js

```diff
@@ -1085,6 +1100,7 @@ $.entwine('ss', function($) {
           iframeId="login-dialog-iframe"
           iframeClassName="login-dialog__body__iframe"
           isOpen={isOpen}
+          onClosed={handleClose}
```

Comment:
Handling the react close button this way is a pattern we do elsewhere, e.g. with the tinymce link modals.